avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -436,6 +458,46 @@ cpdef (double, double) computeEntMassAndForce(double dt, double entrMassCell,
 
   return dm, areaEntrPart
 
+cpdef (double, double) computeDetMassAndForce(double dt, double detCell,
+                                              double areaPart, double uMag):
+  """ compute force component due to detrained mass
```

Comment:
isn't it actually just the detrained mass that is computed? consider updating docstring